### DISCUSSION TITLE
Friends from Trakt.tv

### DISCUSSION THREAD

**Discussion by joe307bad** [trakt/showly#742] [Ideas / Feature Requests]
Have you considered integrating friends from the Trakt.tv API? https://trakt.docs.apiary.io/#reference/users/friends

**Comment by michaldrabik** [comment]
To be honest the very first initial idea for Showly was to create an app that was free from the social aspects and just focuses on the main purpose. So to answear the question unfortunately not considering adding friends and other social aspects. 

For now.

**Reply by joe307bad** [reply]
Thanks for the response. If I had some free time to make a simple screen that used this API to display the info provided from Trakt, would that PR have a chance of being merged?

**Reply by michaldrabik** [reply]
Probably not because still the same philosophy applies for this particular thing. 

However you can always create your own fork. Add the features you are missing the most and just keep using the app.

**Reply by joe307bad** [reply]
Understood. Thanks for the response. Love the app and appreciate all the hard work. I recommend it to everyone.
